### PR TITLE
Add compile constant check to use Production appsettings

### PR DESCRIPTION
**Pull request by cyaneko**
Progresses #148. The next step is to add necessary modifications in CI and later CD.

### CODE REVIEW

**Review by cangi75 — APPROVED** [review]

**Review by daveedoo — APPROVED** [review]

**Review by Michalski-Artur — COMMENTED** [review]
All right. So as far as I follow, the idea is to run sql server instance in separate container and then connect to it using localhost connection string. Correct me if I am wrong. And you will be running it on linux, right?

File: Server-Module/Server/appsettings.Production.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "ServerDBContext": "Server=127.0.0.1,1433;Database=ServerDB;Trusted_Connection=True;MultipleActiveResultSets=true",
```
[the remark this answers is not in the document]
Comment (cyaneko) [reply Server-Module/Server/appsettings.Production.json:11]: All right, I will remove them

**Comment by Michalski-Artur** [comment]
Yeah. So probably it won't work with current connection string. Trusted_Connection only works in Windows domain. In order to fix that you should provide credentials to authenticate. Your connection string should match pattern below:
image
Also when running container with sql server instance you would have to add new user

**Comment by cyaneko** [comment]
> Yeah. So probably it won't work with current connection string. Trusted_Connection only works in Windows domain. In order to fix that you should provide credentials to authenticate. Your connection string should match pattern below:
> image
> Also when running container with sql server instance you would have to add new user

I'll keep that in mind while writing CD job files

**Comment by cyaneko** [comment]
This PR now closes #147.

**Review by mtabor99-pw — APPROVED** [review]

**Review by Michalski-Artur — APPROVED** [review]

**Pull request merged**